WordPress/Classes/WordPress.xcdatamodeld/.xccurrentversion

```diff
@@ -3,6 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>_XCCurrentVersionName</key>
-	<string>WordPress 29.xcdatamodel</string>
+	<string>WordPress 30.xcdatamodel</string>
```

Comment:
Since we implemented the incremental migrator we can err on the side of caution and bump the model as often as it changes. There is a unit test for running migration to current from like model 22 *I think*.